resources/migrate-to-swift.md

```diff
@@ -47,10 +47,34 @@ If you are using Cocoapods to manage your Adobe Experience Platform Mobile SDK d
   pod 'AEPUserProfile'
 ```
 
-Save the `Podfile` and run `pod repo update` to update your local Cocoapods repository.
+Save the `Podfile` and run `pod repo update` to update your local CocoaPods repository.
 
 Once the previous command is complete, run `pod install` or `pod update` to update the application dependencies.
 
+### SPM
+
+You can now also use SPM to manage your Adobe Experience Platform Mobile SDK dependencies. To add the AEP Packages to your application, from the Xcode menu select:
+
+```
+File > Swift Packages > Add Package Dependency...
+```
+
+Enter the URL for the AEP package repositories: 
```

Comment:
"Enter the URL for the Platform package repositories:"

Reply:
In both the instances I am referring to the AEP version of the SDKs, so in this context Platform packages is confusing for a developer.
Perhaps we could use `AEP SDK packages`?